### PR TITLE
Fixed mobile view for auth dialog

### LINKED ISSUE #2358: Fix mobile view of signup and login dialog
**Actual Behaviour**

mobile view of signup and login dialog is not proper

**Expected Behaviour**

Fix the mobile view of signup and login dialog

**Screenshots of the issue**

Capture

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2358

Changes: Made mobile responsive for auth dialog ( login, signup and forget password)

Demo Link: https://pr-2385-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
Capture
Capture
Capture

### CODE REVIEW

File: src/components/Auth/ForgotPassword/ForgotPassword.react.js [hunk 1]
```diff
@@ -107,7 +108,9 @@ class ForgotPassword extends Component {
     return (
       <React.Fragment>
         <DialogTitle>
-          <Translate text="Forgot Password ?" />
+          <ForgetText>
```
Comment (akshatnitd) [inline src/components/Auth/ForgotPassword/ForgotPassword.react.js]: Rename it to `ForgotTextContainer`

File: src/components/Auth/AuthStyles.js [hunk 2]
```diff
@@ -2,6 +2,13 @@ import styled, { css } from 'styled-components';
 import _PasswordField from 'material-ui-password-field';
 import _OutlinedInput from '@material-ui/core/OutlinedInput';
 import _Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
```
Comment (akshatnitd) [inline src/components/Auth/AuthStyles.js]: import as _FormControl

File: src/components/Auth/AuthStyles.js [hunk 3]
```diff
@@ -10,11 +17,16 @@ export const breakPointStyle = css`
 `;
 
 export const responsiveInputStyle = css`
+  margin: auto;
   width: 17rem;
   height: 2.1rem;
   ${breakPointStyle};
 `;
 
+export const AuthForm = styled(FormControl)`
```
Comment (akshatnitd) [inline src/components/Auth/AuthStyles.js]: Rename to `FormControl`

**Comment by samagragupta** [comment]
@akshatnitd Made the changes

**Comment by samagragupta** [comment]
@fragm3 Added Screenshots

File: src/components/Auth/AuthStyles.js [hunk 4]
```diff
@@ -2,6 +2,13 @@ import styled, { css } from 'styled-components';
 import _PasswordField from 'material-ui-password-field';
 import _OutlinedInput from '@material-ui/core/OutlinedInput';
 import _Button from '@material-ui/core/Button';
+import _FormControl from '@material-ui/core/FormControl';
+
+export const ForgotTextContainer = styled.div`
```
Comment (fragm3) [inline src/components/Auth/AuthStyles.js]: Not needed.

File: src/components/Auth/AuthStyles.js [hunk 5]
```diff
@@ -10,11 +17,16 @@ export const breakPointStyle = css`
 `;
 
 export const responsiveInputStyle = css`
+  margin: auto;
```
Comment (fragm3) [inline src/components/Auth/AuthStyles.js]: Not required.

File: src/components/Auth/AuthStyles.js (hunk 4 above)
Comment (samagragupta) [reply src/components/Auth/AuthStyles.js]: If it is not applied.
Capture

File: src/components/Auth/AuthStyles.js (hunk 5 above)
Comment (samagragupta) [reply src/components/Auth/AuthStyles.js]: @fragm3 removed that

File: src/components/Auth/AuthStyles.js (hunk 4 above)
Comment (fragm3) [reply src/components/Auth/AuthStyles.js]: @samagragupta No need for going till 320px, we are focussing on newer phones, hence till 375px max.

Comment (akshatnitd) [reply src/components/Auth/AuthStyles.js]: @fragm3 @samagragupta  360px+ should be focussed.

**Comment by samagragupta** [comment]
@fragm3 This is because of removing margin: auto (https://github.com/fossasia/susi.ai/pull/2385#discussion_r298803449)
I have added it again.

File: src/components/Auth/AuthStyles.js (hunk 4 above)
Comment (samagragupta) [reply src/components/Auth/AuthStyles.js]: @akshatnitd @fragm3 Removed that

**Comment by samagragupta** [comment]
@fragm3 Its Fixed

File: src/components/Auth/SignUp/SignUp.react.js [hunk 6]
```diff
@@ -302,7 +303,10 @@ class SignUp extends Component {
           <CloseButton onClick={this.handleDialogClose} />
         </DialogTitle>
         <DialogContent>
-          <FormControl error={emailErrorMessage !== ''}>
+          <FormControl
+            style={{ style: '100%' }}
```
Comment (akshatnitd) [inline src/components/Auth/SignUp/SignUp.react.js]: What is `style: 100%`?

File: src/components/Auth/SignUp/SignUp.react.js [hunk 7]
```diff
@@ -11,7 +11,7 @@ import Recaptcha from 'react-recaptcha';
 import FormHelperText from '@material-ui/core/FormHelperText';
 import DialogTitle from '@material-ui/core/DialogTitle';
 import DialogContent from '@material-ui/core/DialogContent';
-import FormControl from '@material-ui/core/FormControl';
+// import FormControl from '@material-ui/core/FormControl';
```
Comment (akshatnitd) [inline src/components/Auth/SignUp/SignUp.react.js]: Remove comments

File: src/components/Auth/SignUp/SignUp.react.js (hunk 6 above)
Comment (samagragupta) [reply src/components/Auth/SignUp/SignUp.react.js]: Sorry, I have used that for testing.

File: src/components/Auth/SignUp/SignUp.react.js (hunk 7 above)
Comment (samagragupta) [reply src/components/Auth/SignUp/SignUp.react.js]: @akshatnitd Made the changes

**Pull request merged**